src/main/java/com/elixr/poc/service/PurchaseService.java

```diff
@@ -0,0 +1,35 @@
+package com.elixr.poc.service;
+
+import com.elixr.poc.Constant.Constants;
+import com.elixr.poc.Repository.PurchaseRepository;
+import com.elixr.poc.rest.response.DeleteResponse;
+import org.springframework.stereotype.Service;
+
+import java.util.UUID;
+
+@Service
+public class PurchaseService {
+    private final PurchaseRepository purchaseRepository;
+    private DeleteResponse deleteResponse;
+
+    public PurchaseService(PurchaseRepository purchaseRepository) {
+        this.purchaseRepository = purchaseRepository;
+    }
+
+    public DeleteResponse deletePurchaseDetails(UUID purchaseId) {
+        deleteResponse = new DeleteResponse();
+        boolean purchaseRecord = purchaseRepository.existsById(purchaseId);
+        if (purchaseRecord) {
```

Comment:
Naming of boolean variable is not proper.
better name is "purchaseRecordExists"

Reply:
Updated has suggested.